test/test_helper.rb

```diff
@@ -48,6 +49,28 @@ def unstub_project_callbacks
     Project.any_instance.unstub(:clean_repository)
   end
 
+  def map_ar_queries
+    QueryDiet::Logger.queries.map(&:first) - ["select 1"]
+  end
+
+  def ar_queries
```

Comment:
don't need these 1-off methods for now, `ar_queries` can just return the sql array